src/components/NewCommentForm.tsx

```diff
@@ -1,13 +1,18 @@
 import classNames from 'classnames';
 import React, { useState } from 'react';
-import { CommentData } from '../types/Comment';
+import {
+  addComment as apiAddComment,
+} from '../features/commentsSlice';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
 
 type Props = {
-  onSubmit: (data: CommentData) => Promise<void>;
+
 };
```

Comment:
Props type is an empty object. It's not needed if no props are passed to the component.